core/modules/dashboard/includes/block.news.inc

```diff
@@ -40,13 +40,30 @@ class DashboardNewsBlock extends Block {
    */
   public function getContent() {
     $news = $this->getNews();
+    $days_before_news_is_old = 14;
```

Comment:
hm, I'm not sure I like this variable name. How about `$new_period` or `$new_duration`?

Reply:
@jenlampton I'm happy to make this change, but we should then add an inline comment to explain what "new period" or "new duration" is, since that won't be immediately apparent to everyone reading the code.

If we are to add inline comments, and since that variable is not being used anywhere else, I'd be happy to remove it completely, and then change the `$days_before_news_is_old * 24 * 60 * 60` thing a few lines below to a hard-coded `1209600` and add a comment there instead...

Reply:
> If we are to add inline comments, and since that variable is not being used anywhere else, I'd be happy to remove it completely, and then change the $days_before_news_is_old * 24 * 60 * 60 thing a few lines below to a hard-coded 1209600 and add a comment there instead...

I like this idea :)

Reply:
Done: [`f57e8a0` (#3633)](https://github.com/backdrop/backdrop/pull/3633/commits/f57e8a0be11a952070cb749eac46a8d30e621d6f)